client/src/components/providers/storeProviders.js

```diff
@@ -149,19 +150,23 @@ export const StoreProvider = (storeAction, storeGetter) => {
             this.load();
         },
         computed: {
-            ...mapGetters([storeGetter]),
+            ...mapGetters([storeGetter, queryStatsGetter]),
             attributes() {
                 return this.toCamelCase(this.$attrs);
             },
             result() {
                 return this[storeGetter](this.attributes);
             },
+            totalMatches() {
```

Comment:
We should rename `queryStatsGetter` and `totalMatches`, since this is a generic store provider, accordingly the naming should probably also more generic. Maybe `storeGetterCount` and just `count` as parsed slot data parameter?

Reply:
I agree to make the naming more generic, and I have changed them to `storeCountGetter` and `count` respectively for now.
But I think just `count` may be confused with *the number of items returned in payload* which is not this value, this value represents *the total number of items that match the current query parameters without any pagination or limit*.
I'm happy to rename it to something more accurate, I always consider naming one of the hardest parts :laughing: 

Any better ideas?

Reply:
That looks fine to me. Thanks.